openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1228,14 +1230,19 @@ Authorization: Bearer czZCaGRSa3F0MzpnWDFmQmF0M2JW
 
 ## Deferred Credential Response {#deferred-credential-response}
 
-The Deferred Credential Response uses the `credentials` and `notification_id` parameters as defined in (#credential-response).
+A Deferred Credential Response may either contain the requested Credentials or further defer the issuance:
```

Comment:
The response can also indicate that there are no credentials for the holder (for a variety of reasons) and that the wallet should not poll any longer. I think it's worth adding normative description of what this response looks like 

- 204
- 200 with an empty response
- 200 & and empty `credentials` array?

Reply:
I think this is out of scope for this PR. I'll convert it into a new issue instead.